client/client.go

```diff
@@ -15,88 +15,85 @@
 package client
 
 import (
-	"fmt"
-	"net/http"
-	"net/url"
+	"log"
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/client/restclient"
-	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/client/unversioned"
 )
 
-type AppControllerClient struct {
-	*http.Client
-	Root                   *url.URL
-	dependenciesURL        *url.URL
-	resourceDefinitionsURL *url.URL
+type Interface interface {
```

Comment:
IMO too common name. IMO it should be a bit more specific - maybe `AppControllerInterface`?